.eslintrc.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['warn', { code: 120, ignoreRegExpLiterals: true }],
     'vue/attribute-hyphenation': 'off',
+    'vue/component-name-in-template-casing': ['error', 'PascalCase'],
```

Comment:
I added this per a discussion in the MS Teams Developers channel, though I didn't see any different with this rule here and trying some different custom component names. Anyways, this might serve us in the future.

Reply:
Should we also add it in our other repos to keep things consistent or no need?

Reply:
Yeah, probably, even though it has not effect today (possibly an eslint version thing). I'll update Filings UI and Edit UI...

Reply:
We can create a techdebt ticket for them if you want Sev. Or we can just add this when we're doing some other work there. Nothing urgent.

Reply:
Done.

Reply:
Awesome, thanks 🙌